<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="MSI" failures="1" tests="3" name="TestNGExample" time="0.174" errors="0" timestamp="2023-01-14T07:40:01 IST">
  <testcase classname="com.automation.test.CartPageTest" name="verifyUserCanAddItemIntoCart" time="0.002">
    <failure type="java.lang.RuntimeException" message="Test Failed">
      <![CDATA[java.lang.RuntimeException: Test Failed
at com.automation.test.CartPageTest.verifyUserCanAddItemIntoCart(CartPageTest.java:10)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)
at com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:109)
... Removed 24 stack frames]]>
    </failure>
  </testcase> <!-- verifyUserCanAddItemIntoCart -->
  <testcase classname="com.automation.test.CartPageTest" name="verifyUserCanChangeQuantityOfProductOnCartPage" time="0.0">
    <skipped/>
  </testcase> <!-- verifyUserCanChangeQuantityOfProductOnCartPage -->
  <testcase classname="com.automation.test.CartPageTest" name="verifyUserCanRemoveItemFromCart" time="0.001">
    <skipped/>
  </testcase> <!-- verifyUserCanRemoveItemFromCart -->
</testsuite> <!-- TestNGExample -->
